scripts/validate-schema.js

```diff
@@ -153,6 +153,12 @@ function removeFields(obj) {
   } else {
     throw "SPDX or licenseURL fields missing";
   }
+  if (Object.prototype.hasOwnProperty.call(obj, "url")) {
```

Comment:
because `repositories.url` becomes `url`, `url` will be at the root of the schema, so we catch this by forcing it into `repositories`. Since `repositories.name` becomes `name` and conflicts with the existing `name` (of the digital solution), we don't alter it. See the handling of `license` a few lines above.